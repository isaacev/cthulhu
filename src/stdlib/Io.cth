use ::Kernel::*;

#[inline]
fn println(str: Str) -> () {
  print(str ++ "\n");
}
