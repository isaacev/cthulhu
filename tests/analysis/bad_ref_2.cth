use ::Io;

fn main() -> () {
  Io::doesNotExist();
}
