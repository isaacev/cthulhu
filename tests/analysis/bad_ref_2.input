use IO;

fn main() -> Void {
  IO::doesNotExist();
}
