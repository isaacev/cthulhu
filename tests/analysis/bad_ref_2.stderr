
  UNKNOWN FIELD  -------------------------------------------------------------

  Reference to an unknown field in a namespace.

    : bad_ref_2
  4 |   Io::doesNotExist();
    |       ^^^^^^^^^^^^

